build_daemon/test/daemon_test.dart

```diff
@@ -194,10 +194,15 @@ Future<Process> _runDaemon(var workspace, {int timeout = 30}) async {
     }
       ''').create();
 
-  var packageArg = '--packages=${(await Isolate.packageConfig).path}';
-
-  var process = await Process.start(
-      Platform.resolvedExecutable, [packageArg, 'test.dart'],
+  var args = [
+    ...Platform.executableArguments,
+    if (Platform.executableArguments
+            .indexWhere((arg) => arg.startsWith('--packages=')) ==
```

Comment:
This should just check for `--packages`, you can pass `--packages <path>` as well afaik.

Reply:
`Setting VM flags failed: Unrecognized flags: packages`